casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -468,23 +429,20 @@ object RuntimeManager {
       override def storageRepr(hash: StateHash): T[F, Option[String]] =
         runtimeManager.storageRepr(hash).liftM[T]
 
-      override def computeBonds(hash: StateHash): T[F, scala.Seq[Bond]] =
+      override def computeBonds(hash: StateHash): T[F, Seq[Bond]] =
         runtimeManager.computeBonds(hash).liftM[T]
 
-      override def computeBalance(start: StateHash)(user: ByteString): T[F, Long] =
-        runtimeManager.computeBalance(start)(user).liftM[T]
-
       override def computeDeployPayment(
           start: StateHash
       )(user: ByteString, amount: Long): T[F, StateHash] =
         runtimeManager.computeDeployPayment(start)(user, amount).liftM[T]
 
-      override def getData(hash: StateHash)(channel: Par): T[F, scala.Seq[Par]] =
+      override def getData(hash: StateHash)(channel: Par): T[F, Seq[Par]] =
         runtimeManager.getData(hash)(channel).liftM[T]
 
       override def getContinuation(
           hash: StateHash
-      )(channels: Seq[Par]): T[F, scala.Seq[(scala.Seq[BindPattern], Par)]] =
```

Comment:
what's this, what's this? it's magic in the air...

please tell me it's the default Seq used.

Reply:
WDYM? I hope so too. Lemme see. Yup, there's no imports from `scala.collection` in the file, so it's those `scala.` prefixes were just redundant.